### ISSUE TITLE
Can not update Contract

### ISSUE THREAD

**Issue by georgeserbezov** [MapCreatorEU/api-wrapper#86]
When try to update contract date, the request looks like that: 
```
{"date_start":"2017-11-05T00:00:00.000Z","date_end":"2017-11-18T00:00:00.000Z","dateStart":"2017-11-05T00:00:00.000Z","dateEnd":"2017-11-18T00:00:00.000Z"}
```

**Comment by Mechazawa** [comment]
did you upgrade to the latest version of the wrapper?

**Comment by Mechazawa** [comment]
Should be fixed in 7d332b341c684e2fc90d3b87e992781fb47725b8

**Comment by beschoenen** [comment]
Works
